web-wallet/src/routes/(welcome)/+layout.svelte

```diff
@@ -13,5 +23,51 @@
     height="31"
   />
 </header>
+{#key retryKey}
+  {#await networkStore.connect().then(() => networkStore.init())}
+    <div class="welcome-layout__loading">
+      <Throbber />
+      <strong>CONNECTING TO THE NETWORK</strong>
+    </div>
+    <!-- eslint-disable-next-line no-unused-vars -->
+  {:then _}
```

Comment:
```suggestion
  {:then}
```

Reply:
I thought it wasn't a legit syntax as I haven't seen mention of it in the svelte docs and for some reason my editor was showing me an error; but you're right: thanks for spotting that!